lms/djangoapps/support/urls.py

```diff
@@ -7,6 +7,7 @@
 from support.views.certificate import CertificatesSupportView
 from support.views.course_entitlements import EntitlementSupportView
 from support.views.enrollments import EnrollmentSupportListView, EnrollmentSupportView
+from support.views.course_duration import CourseDurationSupportView
```

Comment:
General comment: I think we should refer to this as "Feature based enrollments"

Duration is only half of it, the other half is content type gating.